src/cpp/session/resources/schema/user-prefs-schema.json

```
@@ -1054,6 +1054,18 @@
             "title": "Double click to select in the Console",
             "description": "Whether double-clicking should select a word in the Console pane."
         },
+        "console_suspend_blocked_notice": {
+            "type": "boolean",
+            "default": false,
```

```suggestion
            "default": true,
```

I think this should default to _true_ to maximize exposure during development. We can switch the default to false later if we need to based on feedback.